compiler-rt/lib/scudo/standalone/vector.h

```diff
@@ -35,7 +35,9 @@ template <typename T> class VectorNoCtor {
     DCHECK_LE(Size, capacity());
     if (Size == capacity()) {
       const uptr NewCapacity = roundUpPowerOfTwo(Size + 1);
-      reallocate(NewCapacity);
+      if (!reallocate(NewCapacity)) {
+        return;
+      }
```

Comment:
Can we have another function like `try_push_back` to support this behavior and only silence the on that path? And/Or we can specialize the Vector<char> to use `try_push_back` so that other OOM usage on other Vector instantiations can still be detected.

Reply:
Just came up with another idea, we can have something like StaticVector which uses a local array and never grows the buffer. So that in an OOM situation, we don't even bother allocating a single page for error message (given the error messages are short in general)

Reply:
There aren't any other uses of Vector, which is why I left it like this. I do agree that having a fixed string is a good idea. Would you mind if I did that as a follow-up rather than change this one? I think that will be a slightly bigger change since I need to tear up the ScopedString a bit to do it right.